modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -904,6 +904,14 @@ const register = (editor: Editor): void => {
     default: true
   });
 
+  registerOption('current_user_id', {
```

Comment:
Arguably the word `current` is superfluous
```suggestion
  registerOption('user_id', {
```

Reply:
Done!